hack/build-release.sh

```diff
@@ -4,7 +4,10 @@ set -e -x -u
 
 mkdir -p tmp/
 
-ytt -f config/ -f config-release | kbld -f- > ./tmp/release.yml
+# makes the get_kappctrl_ver function available (scrapes version from git tag)
+source $(dirname "$0")/version-me.sh
```

Comment:
nit: version-me.sh -> version-utils.sh?

Reply:
:)  imperative "version me" vs. declarative "these are the version utils" ,  eh?   np.